File: C/1041/1041.c
#include <stdio.h>

int main()
{
    float a,b;
    scanf("%f %f",&a,&b);
    if(a*b > 0)
    {
        if(a < 0)
            puts("Q3");
        else
            puts("Q1");
    }
    else if(a*b < 0)
    {
        if(a < 0)
            puts("Q2");
        else
            puts("Q4");
    }
    else
    {
        if(a != 0)
            puts("Eixo X");
        else if(b != 0)
        {
            puts("Eixo Y");
        }
        else
            puts("Origem");
    }
    return 0;
}

File: beecrowd/Python/1041.py
x, y = [float(a) for a in input().split()]

if x * y < 0:
	if x < 0:
		print('Q2')
	else: 
		print('Q4')
elif x * y > 0:
	if x < 0:
		print('Q3')
	else:
		print('Q1')
else:
	if x != 0:
		print('Eixo X')
	elif y != 0:
		print('Eixo Y')
	else:
		print('Origem') 
